changelog.md

```diff
@@ -1,8 +1,10 @@
 # Changelog
 
 
-### 2.4.x
-- Test up to Wordpress 4.7.3
+### 2.4.2
+- Test up to Wordpress 4.8.3
+- Uploadcare widget updated to version 3.x
```

Comment:
this alone calls for major version update: http://semver.org/

Reply:
Not clear for me.
Could you describe in more details?